app/src/main/java/com/framgia/music_40/screen/main/MainActivity.java

```diff
@@ -35,6 +41,10 @@ private void initView() {
         BottomNavigationView bottomNavigationView = findViewById(R.id.navigation_bottom);
         bottomNavigationView.setOnNavigationItemSelectedListener(this);
         mNavigator = new Navigator();
+        mServicePlayMusicManager = new ServicePlayMusicManager();
+        mServicePlayMusicManager.serviceConnect(this, bindIntent(this));
```

Comment:
refactor và xóa từ dòng 23->27
mServicePlayMusicManager.serviceConnect(this, new Intent(context, ServicePlayMusic.class));